using_the_compiler/README.md

```diff
@@ -44,6 +44,18 @@ Make sure to always use `--release` for production-ready executables and when pe
 
 The reason for this is that performance without full optimizations is still pretty good and provides fast compile times, so you can use the `crystal` command almost as if it were an interpreter.
 
+To reduce the binary size, you can add the `--no-debug` flag and use the `strip` command. Debug symbols will be removed, use this options if only the size is an issue and you won't need to debug the program.
+
+## Creating a standalone executable
+
+To build a standalone executable of your program:
+
+`
+$ crystal build some_program.cr --release --static
+```
+
+More informations about statically linking [can be found on the wiki](https://github.com/crystal-lang/crystal/wiki/Static-Linking)
```

Comment:
I think we shouldn't include this link.

Reply:
But if we don't provide further informations about known issues the users will face, they will lose their time searching answers in existing issues, or even create new ones.

Reply:
Honestly the `--static` flag won't do what 99% of users think it does. They'll think static compilation -> go, and our static compilation is nothing like go's because we still link to non-crystal C libraries, whereas go doesn't. So I strongly vote to just leave the flag out until we have an acceptable static linking story on any target that *isn't* musl.